### PR TITLE
updates and small fixes

### PR DESCRIPTION
**Pull request by Leon-Bichmann**
# nf-core/mhcquant pull request

Many thanks for contributing to nf-core/mhcquant!

Please fill in the appropriate checklist below (delete whatever is not relevant).
These are the most common things requested on pull requests (PRs).

## PR checklist

- [ ] This comment contains a description of changes (with reason)
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If necessary, also make a PR on the [nf-core/mhcquant branch on the nf-core/test-datasets repo](https://github.com/nf-core/test-datasets/pull/new/nf-core/mhcquant)
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Documentation in `docs` is updated
- [ ] `CHANGELOG.md` is updated
- [ ] `README.md` is updated

**Learn more about contributing:** [CONTRIBUTING.md](https://github.com/nf-core/mhcquant/tree/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -9,8 +9,6 @@
   * [Reproducibility](#reproducibility)
 * [Main arguments](#main-arguments)
   * [`--sample_sheet`](#--sample_sheet)
```
Comment (ggabernet) [inline docs/usage.md:11]: There is a kind of convention in `nf-core` now, that the input will be given with the parameter `--input`

Comment (Leon-Bichmann) [reply docs/usage.md]: ok docs are updated

**Comment by Zethson** [comment]
So input is now always a sample sheet?

I thought of:

```
input <- raw or mzml
sample_sheet <- sample sheet
```
Your solution works of course, but may be annoying for some users who don't have a complicated experimental design?

But you know best of course. If this is not an issue then +1

**Comment by Leon-Bichmann** [comment]
ok if someone approves ill merge

**Pull request merged**